app/controllers/users_controller.rb

```diff
@@ -47,6 +47,20 @@ def destroy
     redirect_to users_url
   end
 
+  def following
+    @title = "Following"
+    @user = User.find_by params[:id]
```

Comment:
plz, remove this line, because this file have `before_action :load_user, except: %i(index new create)` code